tournament/schedule.go

```diff
@@ -33,8 +34,7 @@ func BuildRandomSchedule(teams []model.Team, scheduleBlocks []model.ScheduleBloc
 	// Adjust the number of matches to remove any excess from non-perfect block scheduling.
 	numMatches = int(math.Ceil(float64(numTeams) * float64(matchesPerTeam) / TeamsPerMatch))
 
-	file, err := os.Open(fmt.Sprintf("%s/%d_%d.csv", filepath.Join(model.BaseDir, schedulesDir), numTeams,
-		matchesPerTeam))
+	file, err := os.Open(fmt.Sprintf("%s/%d_14.csv", filepath.Join(model.BaseDir, schedulesDir), numTeams))
```

Comment:
Agreed, "14" isn't very helpful here, since it's just for two-alliance.  I've renamed this.  

We only support exactly two schedule files--12_two_alliance and 13_two_alliance.  The other ones are 3-way-alliance games and we aren't using them.  Also, this breaks for anything that isn't 12 or 13 way games, but that's as-designed.